docs/general/getting-started.md

```diff
@@ -13,17 +13,22 @@ The Polkadot Wiki serves as the central source of truth for Polkadot. It is a co
 initiative led by Web3 Foundation to keep an up-to-date resource on the best information for
 learning, building, and maintaining on Polkadot.
 
-The Wiki has three main sections: Learn (for learners), Build (for people that want to build on
-Polkadot), and Maintain (for people that want to maintain the network).
+The Wiki has three main sections: **Learn** (for learners), **Build** (for people that want to build
+on Polkadot), and **Maintain** (for people that want to maintain the network).
 
 :::
 
 ## What is Polkadot?
 
-Polkadot is the first fully-sharded blockchain.
+See the video below made by [DeFi Teller](https://defiteller.guru/) to know more about Polkadot.
```

Comment:
The videos (and third-party content) are typically placed at the end of the sections.

Reply:
I thought it was more catchy at the start of the section, I moved it down.

Reply:
Yeah, they did a great job with the visuals. The Wiki docs are sort of "text first" resources, and videos should serve the purpose of augmenting the text content.